public/app/dashboard/profile.html

```diff
@@ -1,39 +1,61 @@
 <div class="profile-container">
-
   <div class="profile-summary-container">
-    <div class="profile-summary-column profile">
-      <div>
-        <img class="profile-summary-avatar" data-ng-src="{{user.canvas_image}}" data-ng-if="user.canvas_image"/>
+    <div class="profile-summary-column">
+      <img class="profile-summary-avatar" data-ng-src="{{user.canvas_image}}" data-ng-if="user.canvas_image"/>
+      <div class="profile-engagement-avatar-caption">
         <a>Edit Profile</a>
       </div>
-      <div>
-        <h2 data-ng-bind="user.canvas_full_name"></h2>
-        <div>Write4Change</div>
-        <div>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-          eiusmod tempor incididunt ut labore et dolore magna aliqua.
-        </div>
-        <div>
-          <a>#photography</a> <a>#camping</a> <a>#travel</a>
+    </div>
+    <div class="profile-summary-column">
+      <div class="profile-summary-banner">
+        <h1 data-ng-bind="user.canvas_full_name"></h1>
+        <div class="profile-summary-user-search">
+          <form class="form-inline search-form" name="userSearchForm" data-ng-submit="search()">
+            <label for="profile-user-search" class="sr-only">Search for users</label>
+            <div class="input-group search-container">
+              <input type="text" id="profile-user-search" class="form-control" placeholder="Search for other people" data-ng-model="userToSearch">
+              <span class="input-group-btn">
+                <button type="submit" class="btn btn-default" title="Search">
+                  <i class="fa fa-search">
+                    <span class="sr-only">Search</span>
+                  </i>
+                </button>
+              </span>
+            </div>
+          </form>
         </div>
       </div>
+      <div class="profile-summary-location">
+        Write4Change (St. Paul, Minnesota, USA)
+      </div>
+      <div>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+        eiusmod tempor incididunt ut labore et dolore magna aliqua.
+      </div>
+      <div>
+        <span data-ng-repeat="hashtag in user.hashtags">
+          <a tool-href data-tool="assetlibrary" data-id="hashtag" data-referring-tool="dashboard" data-referring-id="user.id" data-course="me.course" data-ng-bind="hashtag"></a>
+        </span>
```

Comment:
What happens when you click on hashtag (above) will change. See future dev task: https://jira.ets.berkeley.edu/jira/browse/COL-766